cluster-api/src/main/java/io/scalecube/cluster/Member.java

```diff
@@ -34,6 +45,10 @@ public Address address() {
     return address;
   }
 
+  private static String generateId() {
+    return Long.toHexString(UUID.randomUUID().getMostSignificantBits() & Long.MAX_VALUE);
```

Comment:
So if it's a long value why do we need a string? For back compatibility?

Reply:
Yes, since everywhere it's already referenced as `String`